### ISSUE TITLE
possible to sort before templating?

### ISSUE THREAD

**Issue by cburroughs** [zuazo/ssh_authorized_keys-cookbook#2]
It would be nice if:

``` ruby
ssh_authorize_key 'bob@acme.com' do
  key 'AAAAB3NzaC1yc2EAAAADAQABAAABAQCctNyRouVDhzjiP[...]'
  user 'root'
end

ssh_authorize_key 'alice@acme.com' do
  key 'AAAAB3NzaC1yc2EAAAADAQABAAABAQCySLKbpFRGCrKU/[...]'
  user 'alice'
end
```

and

``` ruby
ssh_authorize_key 'alice@acme.com' do
  key 'AAAAB3NzaC1yc2EAAAADAQABAAABAQCySLKbpFRGCrKU/[...]'
  user 'alice'
end

ssh_authorize_key 'bob@acme.com' do
  key 'AAAAB3NzaC1yc2EAAAADAQABAAABAQCctNyRouVDhzjiP[...]'
  user 'root'
end
```

produced a consistent identical result (by sorting on a key).  That way any upstream in-stability in resource ordering does not result in the template needlessly being re-written.

**Comment by zuazo** [comment]
@cburroughs, thanks for the idea. Implemented in master. I'll let you know when this is released.

**Comment by zuazo** [comment]
Released in [`0.2.0`](https://github.com/onddo/ssh_authorized_keys-cookbook/releases/tag/0.2.0).
